tutorials/deploy-xenforo-to-compute-engine/index.md

```diff
@@ -0,0 +1,267 @@
+
+5. Download your copy of XenForo from XenForo’s website. For this instruction you need to use XenForo 1.5.
+
+## Understanding the Architecture
+
+XenForo needs to work with a web server software and a MySQL database backend to serve requests from the Internet. In this tutorial you will use [Apache HTTP server](https://httpd.apache.org/) for web server software and [MariaDB](https://mariadb.org/) for database backend. At a basic level, the architecture of the project looks as follows:
+
+![Understanding the Architecture](/tutorials/deploy-xenforo-to-compute-engine/architecture_overview.png)
+
+## Installing the Prerequisites
+
+You need to connect to your Compute Engine instance to complete the following steps. To connect from your browser, go to the [VM Instances page](https://console.cloud.google.com/compute/instances) and click **SSH** in the row of the instance that you want to connect to. For other ways of connecting to instances, refer to the [Connecting to Linux Instances](https://cloud.google.com/compute/docs/instances/connecting-to-instance) page of Compute Engine documentation.
+
+1. Update package lists to get information on the newest packages and their dependencies:
+
+	```bash
+	sudo apt-get update
+	```
+
+2. Install Apache server and its module for PHP support:
+
+	```bash
+	sudo apt-get install apache2 apache2-mod-php7.0 -y
+	```
+
+	After the installation is completed, open your browser and visit http://`[SERVER_EXTERNAL_IP]` (note that it is a HTTP link instead of a HTTPS link). `[SERVER_EXTERNAL_IP]` can be found on the [VM Instances](https://console.cloud.google.com/compute/instances) page, in the row of the instance that you connect to. You should see a page titled “Apache Debian Default Page” in your browser. 
+
+3. Install MariaDB server:
+
+	```bash
+	sudo apt-get install mariadb-server -y
+	```
+
+4. Set up MariaDB:
+
+	```bash
+	sudo mysql_secure_installation
+	```
+
+	The script prepares MariaDB for production use. For security reasons you should follow the prompts on screen, set up a root password, remove all anonymous users, disable remote login for root user, remove test database and reload the privilege tables.
+
+5. Create a database and a database user for XenForo to use:
+
+	```bash
+	sudo mysql
+	```
+
+	Then enter the following commands in MariaDB monitor:
+
+	```sql
+	CREATE DATABASE [DATABASE_NAME];
+	CREATE USER ‘[DATABASE_USER]’@’localhost’ IDENTIFIED BY ‘[DATABASE_PASSWORD]’;
+	GRANT ALL PRIVILEGES ON *.* TO ‘[DATABASE_USER]’@’localhost’;
+	```
+
+	Commands above create a new database and a new database user with full privileges in the system. Replace `[DATABASE_NAME]`, `[DATABASE_USER]` and `[DATABASE_PASSWORD]` with values of your own. You may want to write down those values as you need to use them later when installing XenForo.
```

Comment:
"There commands create a new database and a new database user with full privileges in the system. Replace `[DATABASE_NAME]`, `[DATABASE_USER]` and `[DATABASE_PASSWORD]` with your own values. You might want to write down these values, as you need them to install XenForo."